pos_picking_delayed/static/description/index.html

```diff
@@ -3,7 +3,7 @@
 <html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
 <head>
 <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
-<meta name="generator" content="Docutils 0.15.1: http://docutils.sourceforge.net/" />
+<meta name="generator" content="Docutils 0.14: http://docutils.sourceforge.net/" />
```

Comment:
Avoid this change updating your docutils library

Reply:
AFAIK, it will be erased when merging.
@chienandalu when i run readme generator i add --no-gen-html to avoid such noice.

Reply:
@legalsylvain well, that will leave the bot to do it, but we prefer to keep all in sync in one commit.

Reply:
Ok, fixed installing the right versions of docutils and maintainer-tools